src/contracts/core/PaymentCoordinatorStorage.sol

```diff
@@ -0,0 +1,107 @@
+ */
+abstract contract PaymentCoordinatorStorage is IPaymentCoordinator {
+
+    /*******************************************************************************
+                               CONSTANTS AND IMMUTABLES 
+    *******************************************************************************/
+
+    /// @notice The EIP-712 typehash for the contract's domain
+    bytes32 public constant DOMAIN_TYPEHASH =
+        keccak256("EIP712Domain(string name,uint256 chainId,address verifyingContract)");
+    uint32 public immutable MAX_PAYMENT_DURATION;
+    /// @notice max amount of time that a payment can start in the past
+    uint32 public immutable MAX_RETROACTIVE_LENGTH;
+    /// @notice max amount of time that a payment can start in the future
+    uint32 public immutable MAX_FUTURE_LENGTH;
+    /// @notice absolute min timestamp that a payment can start at
+    uint32 public immutable GENESIS_PAYMENT_TIMESTAMP;
+
+    /// @notice The elegationManager contract for EigenLayer
+    IDelegationManager public immutable delegationManager;
+
+    /// @notice The StrategyManager contract for EigenLayer
+    IStrategyManager public immutable strategyManager;
+
+    /*******************************************************************************
+                                       STORAGE 
+    *******************************************************************************/
+
+    /**
+     * @notice Original EIP-712 Domain separator for this contract.
+     * @dev The domain separator may change in the event of a fork that modifies the ChainID.
+     * Use the getter function `domainSeparator` to get the current domain separator for this contract.
+     */
+    bytes32 internal _DOMAIN_SEPARATOR;
+
+    /// @notice list of roots submitted by the paymentUpdater
+    DistributionRoot[] public distributionRoots;
+
+    /// Slot 3
+    /// @notice The address of the entity that can update the contract with new merkle roots
+    address public paymentUpdater;
+    /// @notice The interval in seconds at which the calculation for range payment distribution is done.
+    /// @dev Payment durations must be multiples of this interval.
+    uint32 public calculationIntervalSeconds;
+    /// @notice Delay in timestamp before a posted root can be claimed against
+    uint32 public activationDelay;
+    /// @notice Timestamp for last submitted 
+    uint32 public currPaymentCalculationEndTimestamp;
+
+    /// Slot 4
+    /// @notice the commission for all operators across all avss
+    uint16 public globalOperatorCommissionBips;
+
+    /// @notice Mapping: earner => the address of the entity to which new payments are directed on behalf of the earner
+    mapping(address => address) public claimerFor;
+
+    /// @notice Mapping: claimer => token => total amount claimed
+    mapping(address => mapping(IERC20 => uint256)) public cumulativeClaimed;
+
+    // /// @notice Timestamp for last submitted 
+    // uint32 public currPaymentCalculationEndTimestamp;
```

Comment:
Can be removed

Reply:
[741ef4f](https://github.com/Layr-Labs/eigenlayer-contracts/pull/514/commits/741ef4f93c2a564b07e77ca97ac595e5eb1436c1)